plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1112,11 +1116,86 @@ static int dump_queues(int fd, struct kfd_ioctl_criu_process_info_args *info_arg
 	return ret;
 }
 
+static int dump_events(int fd,
+			struct kfd_ioctl_criu_process_info_args *info_args,
+			CriuKfd *e)
+{
+	int ret = 0, i;
+	struct kfd_ioctl_criu_dumper_args args = {0};
+	struct kfd_criu_event_bucket *ev_buckets;
+	uint8_t *priv_data;
+
+	pr_debug("Dumping %d events\n", info_args->total_events);
```

Comment:
If the changes to plugin_log_msg are introduced in a later patch, better to bring it early and use it everywhere intead of pr_debug/info.

Reply:
These prints can be useful for non-developers to report error messages. They should not be compiled out. plugin_log_msg should only be for debug messages during development as they get compiled out without -DDEBUG